samples/app/unsplash-client/src/main/AndroidManifest.xml

```diff
@@ -14,6 +14,7 @@
         android:theme="@style/Theme.MyApp">
         <activity
             android:name=".RootActivity"
+            android:exported="true"
```

Comment:
`false`

Reply:
https://developer.android.com/about/versions/12/behavior-changes-12#exported

Reply:
All Launcher activities should have `exported = true` from Android 12